src/main/java/net/atomique/ksar/ui/NaturalComparator.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 The kSAR Project. All rights reserved.
+ * Copyright 2022 The kSAR Project. All rights reserved.
```

Comment:
I'm not a lawyer, but I think the copyright should claim the **first** year when the file was developed.

In other words, it claims that "this file has been developed in 2018 by kSar team".

If you update 2018 to 2022, then someone might claim they invented the class in 2019, and we stole from them..

So I would suggest two options:
a) Just keep the very first year
b) Mention all years (e.g. 2018-2022 or 2018, 2022)

Reply:
agree